functions/tech-support

```diff
@@ -68,241 +70,256 @@ fi
 export PATH=/sbin:/usr/sbin:$PATH
 
 header Show Tech-Support  
+  
 header CONFIGURATION 
+  
+function show_version { 
+  if cli-shell-api validateTmplPath entitlement; then
+    header Entitlement
+    show entitlement
+  fi
 
-if cli-shell-api validateTmplPath entitlement; then
```

Comment:
Mixing formatting changes with logic changes is a very bad style. How is the reviewer supposed to follow the logic changes when the file is littered with inconsequential formatting changes?

Reply:
Delete this block